tuptime: add page (#22431)
tldr-pages/tldr: pages/common/tuptime.md

# tuptime

> Report historical and statistical real time of the system, preserving it between restarts.
> Like `uptime`, but extended with startups count, graceful and bad shutdowns, uptime/downtime percentages, and averages since first boot.
> More information: <https://github.com/rfmoz/tuptime>.

- Show system uptime history and statistics:

`tuptime`

> - Enumerate system life as a table:

Reviewer: ```suggestion
- Show system lifetime as table:
```

`tuptime --table`

> - Enumerate system life as a list:

Reviewer: ```suggestion
- Show system lifetime as list:
```

`tuptime --list`

> - Show table with kernel version and boot identifier columns:

Reviewer: ```suggestion
- Show table with kernel version and boot ID:
```

`tuptime --table --kernel --bootid`

> - Limit the report to a range of startup numbers:

Reviewer: ```suggestion
- Report from startup range:
```
IMHO more concise and on point.

> `tuptime --table --since {{first_startup}} --until {{last_startup}}`

Reviewer: ```suggestion
`tuptime --table --since {{1}} --until {{5}}`
```
Concrete examples preferred

- Limit the report from a given number of seconds ago (e.g. one year = 31557600):

> `tuptime --tsince -{{seconds}}`

Reviewer: ```suggestion
`tuptime --tsince -{{31557600}}`
```
A real example maybe? That's one year in seconds.
